ui/raidboss/data/07-dt/trial/byakko-un.ts

```diff
@@ -0,0 +1,421 @@
+      condition: (data, matches) => data.role === 'tank' && matches.target !== data.me,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Provoke Boss',
+          de: 'Boss abspotten',
+          fr: 'Provoquez le Boss',
+          ja: '挑発',
+          cn: '挑衅',
+          ko: '보스 도발',
+        },
+      },
+    },
+    {
+      id: 'ByakkoUn State Of Shock Tank 2',
+      type: 'StartsUsing',
+      netRegex: { id: '9C01', source: 'Byakko' },
+      condition: (data, matches) => data.role === 'tank' && matches.target === data.me,
+      delaySeconds: 12,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Provoke Boss',
+          de: 'Boss abspotten',
+          fr: 'Provoquez le Boss',
+          ja: '挑発',
+          cn: '挑衅',
+          ko: '보스 도발',
+        },
+      },
+    },
+    {
+      id: 'ByakkoUn Roar Counter',
+      type: 'StartsUsing',
+      netRegex: { id: '9C19', source: 'Hakutei', capture: false },
+      run: (data) => {
+        data.roarCount = (data.roarCount ?? 0) + 1;
+      },
+    },
+    {
+      id: 'ByakkoUn Roar Of Thunder',
+      type: 'StartsUsing',
+      netRegex: { id: '9C19', source: 'Hakutei', capture: false },
+      delaySeconds: 14,
+      alarmText: (data, _matches, output) => {
+        if (data.roarCount !== 2)
+          return;
+
+        if (data.role === 'tank' || data.job === 'BLU')
+          return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Tank LB NOW',
+          de: 'JETZT Tank LB',
+          fr: 'Transcendance Tank maintenant !',
+          ja: '今タンクLB',
+          cn: '坦克LB',
+          ko: '탱리밋',
+        },
+      },
+    },
+    {
+      id: 'ByakkoUn Gale Force', // Role spreads + expanding hemispheres
+      type: 'HeadMarker',
+      netRegex: { id: '0065' },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Drop bubble outside',
+          de: 'Blase außen ablegen',
+          fr: 'Déposez la bulle à l\'extérieur',
+          ja: '外にマーカーを置く',
+          cn: '边缘放点名',
+          ko: '장판 바깥에 깔기',
+        },
+      },
+    },
+    {
+      id: 'ByakkoUn Ominous Wind',
+      type: 'GainsEffect',
+      netRegex: { effectId: '5C9' },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Pink bubble',
+          de: 'Pinke Blase',
+          fr: 'Bulle violette',
+          ja: '祟り目',
+          cn: '和其他泡泡散开',
+          ko: '핑크 구슬',
+        },
+      },
+    },
+    {
+      id: 'ByakkoUn Aratama', // Prey marker + 3x puddles
+      type: 'HeadMarker',
+      netRegex: { id: '0004' },
+      condition: Conditions.targetIsYou(),
+      alarmText: (_data, _matches, output) => output.text!(),
```

Comment:
```suggestion
      alertText: (_data, _matches, output) => output.text!(),
```

Reply:
This one I think is worth leaving as an alarm, since if the puddles are dropped poorly they will cause wipes. I don't feel super strongly about that, but I do think there's some merit to having this be alarmed.